drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/ChecksRepeatingCommand.java

```diff
@@ -14,15 +14,14 @@
  * limitations under the License.
  */
 
-package org.drools.workbench.services.verifier.api.client.checks.base;
+package org.drools.workbench.services.verifier.core.checks.base;
 
 import java.util.ArrayList;
 import java.util.Set;
 
 import com.google.gwt.user.client.Command;
 import org.drools.workbench.services.verifier.api.client.CancellableRepeatingCommand;
-import org.drools.workbench.services.verifier.api.client.ParameterizedCommand;
-import org.drools.workbench.services.verifier.api.client.Status;
+import org.drools.workbench.services.verifier.api.client.StatusUpdate;
 
 /**
  * This class handles processing the analysis itself. It supports "batched" processing
```

Comment:
Does the Web Worker still work "in batch" or is it now fired and forgotten (since it's now async?)